monitoring/rid_qualifier/README.md

```diff
@@ -6,6 +6,10 @@ This directory contains a series of tests for qualifying Network Remote ID compl
 
 2. **Aircraft state replayer** (`aircraft_state_replayer.py`): Using the flight data generated by the previous tool, we simulate flights and "fly" them by emitting aircraft states to be fed to the Remote ID Service Provider test harnesses. Specific details and metadata about the flights are included when submitting to the SP test harnesses via the injection API.
 
-3. **Test executor**:  The test executor is  the main routine of `rid_qualifier`: it instructs the `aircraft_state_replayer` to send test data into each SP's injection harness, and then asks the `display_data_evaluator` to query the RID system for expected data and evaluate the query results against expectations to produce qualification results.
+3. **Test executor**: (`test_executor.py`) The test executor is  the main routine of `rid_qualifier`: it instructs the `aircraft_state_replayer` to send test data into each SP's injection harness, and then asks the `display_data_evaluator` to query the RID system for expected data and evaluate the query results against expectations to produce qualification results.
 
-4. **Display Data Evaluator**: Once test aircraft state data is sent to each Remote ID Service Provider's injection test harness to be injected into their systems, the state of the RID system must be queried and then the results of those queries compared against expected results.  This module performs the latter tasks of querying the RID system through each specified Remote ID Display Provider's ingestion test harness, and comparing those query results against expectations. [Flight Blender](https://github.com/openskies-sh/flight-blender) is an open-source Remote ID Display Provider that will implement an ingestion test harness, and therefore can be used to test compliance.
\ No newline at end of file
+4. **Display Data Evaluator**: Once test aircraft state data is sent to each Remote ID Service Provider's injection test harness to be injected into their systems, the state of the RID system must be queried and then the results of those queries compared against expected results.  This module performs the latter tasks of querying the RID system through each specified Remote ID Display Provider's ingestion test harness, and comparing those query results against expectations. [Flight Blender](https://github.com/openskies-sh/flight-blender) is an open-source Remote ID Display Provider that will implement an ingestion test harness, and therefore can be used to test compliance.

```

Comment:
"ingestion" [is now](#498) "observation"

Reply:
I have now changed the text, the link you mention above points to the PR itself.